apps/heft/src/pluginFramework/HeftTaskSession.ts

```diff
@@ -97,11 +97,9 @@ export interface IHeftTaskSession {
   readonly parsedCommandLine: IHeftParsedCommandLine;
 
   /**
-   * The cache folder for the task. This folder is unique for each task, and will not be
-   * cleaned when Heft is run with `--clean`. However, it will be cleaned when Heft is run
-   * with `--clean` and `--clean-cache`.
+   * The cache folder for the task. This is an alias for the temp folder.
    *
-   * @public
+   * @deprecated Use `tempFolderPath` instead.
    */
   readonly cacheFolderPath: string;
```

Comment:
Same here

Reply:
I can't until we do another publish and another cyclic dependency bump.